osu.Game.Rulesets.Osu/Mods/OsuModDifficultyAdjust.cs

```diff
@@ -32,17 +52,21 @@ public class OsuModDifficultyAdjust : ModDifficultyAdjust
 
         protected override void TransferSettings(BeatmapDifficulty difficulty)
         {
-            base.TransferSettings(difficulty);
+            // base.TransferSettings(difficulty);
```

Comment:
Commented code should either be uncommented, or deleted. At current I don't know what the reason for suppressing this is, and it's likely an incorrect thing to do too.

Reply:
Got it.